src/NzbDrone.Core/Configuration/ConfigFileProvider.cs

```diff
@@ -216,6 +223,36 @@ public string UrlBase
             }
         }
 
+        public UseProxy ProxyEnabled
+        {
+            get { return GetValueEnum<UseProxy>("ProxyEnabled", UseProxy.Off); }
+        }
+
+        public ProxyType ProxyType
+        {
+            get { return GetValueEnum<ProxyType>("ProxyType", ProxyType.Http); }
+        }
+
+        public string ProxyHostname
+        {
+            get { return GetValue("ProxyHostname", "localhost"); }
```

Comment:
We use `localhost` in most other cases for a default, but in this case I don't think we should specify a default at all.